service/src/integrationTest/kotlin/fi/espoo/evaka/invoicing/PaymentsIntegrationTest.kt

```diff
@@ -366,6 +396,21 @@ class PaymentsIntegrationTest : FullApplicationTest(resetDbBeforeEach = true) {
         }
     }
 
+    private fun deletePaymentDrafts(
+        today: LocalDate,
+        paymentIds: List<PaymentId>,
+        expectedStatus: Int = 200
+    ) {
+        val (_, response, _) =
+            http
+                .post("/payments/delete-drafts")
+                .asUser(financeUser)
+                .withMockedTime(HelsinkiDateTime.of(today, LocalTime.of(10, 0)))
+                .jsonBody(jsonMapper.writeValueAsString(paymentIds))
+                .response()
```

Comment:
Integration tests should use direct controller calls instead of http requests. It's probably not a big refactor to add `@Autowired` controller as a member property to the class and change all http calls to just call the contoroller's functions directly. There are plenty of other test classes that do this already, if you want examples.